### PR TITLE
feat(i18n): convert review changes button to use i18n primitives

### PR DESCRIPTION
**Pull request by jtpetty**
### Description

This PR adds i18n support for the ReviewChangesButton

### CODE REVIEW

File: packages/sanity/src/desk/i18n/resources.ts [hunk 1]
```diff
@@ -116,6 +116,22 @@ const deskLocaleStrings = {
...
+  'status-bar.review-changes-button.status.syncing.text': 'Saving...',
+
+  /** Label for button when status is saved */
+  'status-bar.review-changes-button.status.saved.text': 'Saved!',
+
+  /** Primary text for tooltip for the button */
+  'status-bar.review-changes-button.tooltip.text': 'Review changes',
+
+  /** Text for the secondary text for tooltip for the button */
+  'status-bar.review-changes-button.tooltip.changes-saved': 'Changes saved',
```
Comment (rexxars) [inline packages/sanity/src/desk/i18n/resources.ts:131]: For consistency I guess the `.tooltip` should be at the end?

**Review by rexxars — APPROVED** [review]
As we talked about over PM, maybe the `.tooltip` should be at the end? No strong preferences, but should be consistent at least.

File: packages/sanity/src/desk/i18n/resources.ts (hunk 1 above)
Comment (jtpetty) [reply packages/sanity/src/desk/i18n/resources.ts:131]: maybe? it's the same tool tip, but this tool tip has a primary and secondary text.

**Review by bjoerge — APPROVED** [review]
LGTM

**Pull request merged**